project/GOVERNANCE.md

```diff
@@ -1,12 +1,12 @@
 # Moby project governance
 
-Moby projects are governed by the [Moby Technical Steering Committee (TSC)](https://github.com/moby/tsc).
-See the Moby TSC [charter](https://github.com/moby/tsc/blob/master/README.md) for
-further information on the role of the TSC and procedures for escalation
-of technical issues or concerns.
+Moby is the open-source project used to build Docker, jointly maintained by Docker
+and the community. As a [community project](https://www.docker.com/community/open-source/),
+we abide by a [Code of Conduct](https://github.com/docker/code-of-conduct)
+and define a governance which ensures the project is community driven and open to
+anyone to contribute.
 
-Contact [any Moby TSC member](https://github.com/moby/tsc/blob/master/MEMBERS.md) with your questions/concerns about the governance or a specific technical 
-issue that you feel requires escalation.
+Contact moby@docker.com with any questions/concerns about the project or governance.
```

Comment:
```suggestion
Contact moby@docker.com with any questions/concerns about the project governance.
```

Reply:
Or maybe
```suggestion
Contact moby@docker.com with any questions/concerns about the enforcement of the Code of Conduct
```
assuming most questions can be just asked in public in GitHub Discussions